mods/taskbar-on-top.wh.cpp

```diff
@@ -1135,15 +1155,25 @@ BOOL WINAPI SetWindowPos_Hook(HWND hWnd,
         GetCursorPos(&pt);
 
         HMONITOR monitor = MonitorFromPoint(pt, MONITOR_DEFAULTTONEAREST);
+
+        MONITORINFO monitorInfo = {sizeof(MONITORINFO)};
+        if (GetMonitorInfo(monitor, &monitorInfo)) {
+            // Corrigir o posicionamento horizontal
+            if (X < monitorInfo.rcWork.left) {
+                X = monitorInfo.rcWork.left;
+            } else if (X > monitorInfo.rcWork.right - cx) {
+                X = monitorInfo.rcWork.right - cx;
+            }
+
+            // Corrigir o posicionamento vertical
+            if (Y < monitorInfo.rcWork.top) {
+                Y = monitorInfo.rcWork.top;
+            }
+        }
```

Comment:
Same comment - this change is related to jump lists, not the start menu. Is it necessary?